### PR TITLE
Add documentation for `-XX:[+|-]ShowHiddenFrames`

### PR DESCRIPTION
**Pull request by fengxue-IS**
Fixes: #887 

Signed-off-by: Jack Lu <Jack.S.Lu@ibm.com>

### CODE REVIEW

**Comment by fengxue-IS** [comment]
@pshipton can you please take a look

**Comment by pshipton** [comment]
jenkins doc stage

File: docs/xxshowhiddenframes.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]ShowHiddenFrames
+
+This option enables or disables the display of JVM generated hidden MethodHandle frames in stacktrace.
```
[the remark this answers is not in the document]
Comment (pshipton) [reply docs/xxshowhiddenframes.md:27]: We don't need `JVM` here either.

**Comment by pshipton** [comment]
https://eclipse-openj9.github.io/openj9-docs-staging/894

**Comment by pshipton** [comment]
https://eclipse-openj9.github.io/openj9-docs/version0.31/ needs to be updated as well.
See an older version for examples.

File: docs/xxshowhiddenframes.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (pshipton) [reply docs/xxshowhiddenframes.md:27]: Are there generated frames which are not hidden by this option? If not, `hidden` should be removed here and everywhere else.

**Comment by pshipton** [comment]
jenkins doc stage

**Review by pshipton — APPROVED** [review]

**Pull request merged**